README.rst

```diff
@@ -33,8 +33,7 @@ into the spiking equivalent:
 
     # add time dimension to inputs
     inp = tf.keras.Input((None, 5))
-    # use Keras' TimeDistributed wrapper to apply Dense over time
-    dense = tf.keras.layers.TimeDistributed(tf.keras.layers.Dense(10))(inp)
```

Comment:
I'm actually not opposed to having this, even if it's not strictly necessary. A lot of users will want to be extending this example to e.g. convnets, where `TimeDistributed` is necessary. You could argue that it's not our job to teach users about `TimeDistributed`, since it's a general Keras layer (not part of this repo), but because it's so tied to making things spiking (and a lot of users doing standard convnets will not have seen it), I think it's good to show it somewhere.

I would be OK with taking it out of the README, if the goal there is to keep things simple and show off how easy it is to change a network to spiking. Or I'm fine to change some of the dense layers to conv layers (either in the README, or the example, or both) so that the TimeDistributed is necessary. I'd also be fine to add some other docs section or example to show it in more detail (I know we do have more examples planned for the future, showing e.g. dithering). However, with this PR, `TimeDistributed` isn't mentioned anywhere, which I think is a problem. My preference would be to just keep things as they currently are in `master` for now, and perhaps when we have something showing `TimeDistributed` more specifically we can remove it from some of these other places.

Reply:
Yeah that makes sense. Lets take it out of the README, but leave it in the example.